src/FlaskRTBCTF/ctf/models.py

```diff
@@ -15,8 +16,20 @@ class Machine(db.Model):
     ip = db.Column(db.String(64), nullable=False)
     hardness = db.Column(db.String, nullable=False, default="Easy")
 
+    users = db.relationship(
```

Comment:
don't need this. It should be a one way relationship.

Reply:
one way relationship as in?